cg/services/delivery_message/delivery_message_service.py

```diff
@@ -1,22 +1,41 @@
+from cg.apps.tb import TrailblazerAPI
+from cg.apps.tb.models import TrailblazerAnalysis
+from cg.exc import OrderNotDeliverableError, OrderNotFoundError
 from cg.server.dto.delivery_message.delivery_message_request import (
     DeliveryMessageRequest,
 )
 from cg.server.dto.delivery_message.delivery_message_response import (
     DeliveryMessageResponse,
 )
 from cg.services.delivery_message.utils import get_message, validate_cases
-from cg.store.models import Case
+from cg.store.models import Case, Order
 from cg.store.store import Store
 
 
 class DeliveryMessageService:
-    def __init__(self, store: Store) -> None:
+    def __init__(self, store: Store, trailblazer_api: TrailblazerAPI) -> None:
         self.store = store
+        self.trailblazer_api = trailblazer_api
 
-    def get_delivery_message(
+    def get_cases_message(
         self, delivery_message_request: DeliveryMessageRequest
     ) -> DeliveryMessageResponse:
         case_ids: list[str] = delivery_message_request.case_ids
+        return self._get_delivery_message(case_ids)
+
+    def get_order_message(self, order_id: int) -> DeliveryMessageResponse:
+        order: Order = self.store.get_order_by_id(order_id)
+        if not order:
+            raise OrderNotFoundError(f"Order with ID {order_id} not found.")
```

Comment:
Design as we discussed: we can consider remapping exceptions in the top layer and enriching them with information there instead of spreading it through the code base. Out of scope

Reply:
For another PR